plutus-tx-plugin/test/Plugin/Basic/Spec.hs

```diff
@@ -32,15 +33,15 @@ basic = testNested "Basic" [
   , goldenUEval "ifOptEval" [ifOpt]
   ]
 
-monoId :: CompiledCode PLC.DefaultUni (Integer -> Integer)
+monoId :: CompiledCode PLC.DefaultUni PLC.DefaultFun (Integer -> Integer)
```

Comment:
@michaelpj  Maybe we should have `CompiledCodeDef` or something. Or the other way around:

```haskell
type CompiledCodeIn uni fun a = <...>
type CompiledCode = CompiledCodeIn PLC.DefaultUni PLC.DefaultFun
```

Reply:
Yeah, nobody wants to see this.

Reply:
You can do this if you want otherwise I'll do it at some point (just use a fixed universe inside `CompiledCode`).

Reply:
I thought a bit about this and decided it would be a bad idea to refactor `CompiledCode` in this PR. We only add extensible built-in function here in the most straightforward way, which is adding the `fun` type variable whenever there already exists the `uni` one. Refactoring what `CompiledCode` means is out of scope, so one of us will do that later.